src/core/CMakeLists.txt

```diff
@@ -451,6 +451,10 @@ SET(QGIS_CORE_SRCS
   fieldformatter/qgsvaluemapfieldformatter.cpp
   fieldformatter/qgsvaluerelationfieldformatter.cpp
 
+  geocms/qgsgeocmsconnection.cpp
+  geocms/qgsgeonodeconnection.cpp
+  geocms/qgsgeonoderequest.cpp
```

Comment:
I would suggest to rename the subdirectory from "geocms" to "geonode" to match structure in src/app